stat/testuv/norm.go

```diff
@@ -0,0 +1,77 @@
+package testuv
+
+import (
+	"fmt"
+	"math"
+
+	"gonum.org/v1/gonum/stat"
+)
+
+// NormalSkewTest test whether the skew is different from the normal distribution.
```

Comment:
```
// NormalSkew tests whether the skew is different from the normal distribution.
// It returns the 2-sided p-value for the hypothesis test.
//
// This function tests the null hypothesis that the skewness of population that the
// sample was drawn from is the same as that of a corresponding normal distribution.
//
// The length of a must be at least 8 otherwise NormalSkew will panic.
```